src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -825,3 +825,69 @@
   - name: Get the details for a virtual cluster
     text: az sql virtual-cluster show -g mygroup -n mycluster
 """
+
+helps['sql db classification'] = """
+type: group
+short-summary: Manage sensitivity classifications.
+"""
+
+helps['sql db classification update'] = """
+type: command
+short-summary: Update a columns's sensitivity classification.
+examples:
+  - name: Update sensitivity classification for a given column.
+    text: az sql db classification update -g mygroup -s myserver -n mydb --schema-name dbo --table-name mytable --column-name mycolumn --information-type Name --label-name "Confidential - GDPR"
```

Comment:
Param names should be like `--table`, `--column`. There is no "`-name`" in azure CLI.

Reply:
Although I've seen some --table-name/--schema-name/etc.. in other commands, I will update to --table/--schema/etc..